readme.md

```diff
@@ -125,6 +127,10 @@
 - [DiscordServ](https://discordserv.com/)
 - [Disboard](https://disboard.org/)
 
+### Bot lists:
+- [discordbots.org](https://discordbots.org/) | [server](https://discord.gg/EYHTgJX)
+- [bots.discord.pw](https://bots.discord.pw/) | [server](https://discord.gg/0cDvIgU2voWn4BaD)
+
```

Comment:
Would it be possible to remove the servers, please?

Reply:
done ed3ebed